### DISCUSSION TITLE
Sundays row is indented in Matrix On Time2 (full calendar) sample. How do I build a working Calendar Matrix?

### DISCUSSION THREAD

**Discussion by Denissant** [kurkle/chartjs-chart-matrix#23] [Q&A] [answered]
I'm using your sample Calendar Matrix from [samples/matrix-on-time2.html](https://github.com/kurkle/chartjs-chart-matrix/blob/next/samples/matrix-on-time2.html) and for some reason the last row (Sundays) is indented: 
![matrix_bugged](https://user-images.githubusercontent.com/79437472/119697410-37128d80-be61-11eb-9a30-c8337db5331f.png)

[Here's the online preview](https://htmlpreview.github.io/?https://github.com/kurkle/chartjs-chart-matrix/blob/next/samples/matrix-on-time2.html)
(Btw, tooltips don't work as intended because `tooltips:` key is used on line 80 instead of the newer key `tooltip:`.)

I've tried to fix the positioning by changing X and Y axes configurations but can't find a solution. One thing that partially "fixed" the issue was removing `round: 'week'` from `scales.x.time` (line 99). After that the last row isn't inconsistently indented anymore but now it looks like this: 
![diagonal_bugged_sundays](https://user-images.githubusercontent.com/79437472/119699686-77730b00-be63-11eb-9eb9-a9414df7304b.png)

I think the sample is supposed to work on v1.0.0-rc of `chartsjs-chart-matrix`, so I tried that and some other versions but nothing seems to work. I don't think it's a bug but just an outdated sample, however I'm new to Javascript and hoping to get some help as I've already tried everything I could think of. 

Thanks in advance and thank you for developing this module.

**Comment by kurkle** [comment, marked as answer]
Couple of issues in the sample, I updated it: https://github.com/kurkle/chartjs-chart-matrix/commit/0de5bea0bc140f41ccc4108e77cc6301d21afd2a

**Reply by Denissant** [reply]
Didn't expect such a quick response, thank you!

**Reply by kurkle** [reply]
Migrated rest of the samples to docs and removed the samples dir. That sample is now available there:
https://chartjs-chart-matrix.pages.dev/samples/time.html
